com.claudiocambra.brickbuster.desktop

```diff
@@ -0,0 +1,11 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2021 Claudio Cambra <claudio.cambra@gmail.com>
+[Desktop Entry]
+Name=BrickBuster
+GenericName=A brick-busting game
+Comment=A brick-breaker game, fun and addictive!
+Exec=com.claudiocambra.brickbuster
```

Comment:
I don't think this is right. It should be the name of the executable to run

Reply:
You would be correct, thanks. Changed to brickbuster_linux.x86_64